src/mame/galaxian/galaxian.cpp

```diff
@@ -7882,17 +7902,25 @@ void guttangt_state::guttangts3(machine_config &config)
 
 
 
-void galaxian_state::jumpbug(machine_config &config)
+void galaxian_state::jumpbugbrf(machine_config &config)
 {
-	galaxian_base(config);
+    galaxian_base(config);
```

Comment:
What's changing here and below? Could you run srcclean on this?

Reply:
Fixed!